src/pug/blocks/product-detail/main.pug

```diff
@@ -0,0 +1,62 @@
+mixin productDetailMain
+  +title("Rượu nho năm 1987")
+  h4.product-detail__price 330.000đ
+  .product-detail__meta
+    .product-detail__ratings
+      i.fas.fa-star
+      i.fas.fa-star
+      i.fas.fa-star
+      i.fas.fa-star-half-alt
+      i.far.fa-star
```

Comment:
Tạo 1 mixin riêng và sử dụng vòng lặp để in ra thay vì như này
VD: 
mixin rating(rates)
+rating(4) => in ra 4 sao vàng 1 sao xám
+rating(3) => in ra 3 sao vàng 2 sao xám

Reply:
E fix hết rồi anh ạ